src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -54,11 +54,8 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
         url = getResmokeLogURL(buildID, { raw: true });
         htmlLogURL = getResmokeLogURL(buildID, { html: true });
       }
-      if (logkeeperMetadata) {
-        jobLogsURL = getSpruceJobLogsURL(
-          logkeeperMetadata.task_id,
-          logkeeperMetadata.execution || 0
-        );
+      if (buildID) {
+        jobLogsURL = getJobLogsURL(buildID);
```

Comment:
This won't populate if we have only a build id. I moved this to the end because we should populate it if we have either a (build_id) or (build_id and test_id)